QUEST.tsv

```diff
@@ -3308,8 +3308,8 @@ QUEST_20151224_003315	It is up to you what path you will take in this adventure.
 QUEST_20151224_003316	Bishop Urbonas
 QUEST_20151224_003317	If the orders from the Demon Warrant are achieved, Orsha will no longer be safe.{nl}I hope our lord will make her decisions wisely.{nl}The situation right now is disastrous enough as it is with the emmigration order.{nl}
 QUEST_20151224_003318	The choice is yours.{nl}I am sure the goddesses want you to lead our salvation too.
-QUEST_20151224_003319	What do you need, oh the Revelator who saved Orsha?
-QUEST_20151224_003320	(The writings are smudged from the rain and hard to make out.)
+QUEST_20151224_003319	Revelator, the one who saved Orsha, what can I do for you?
+QUEST_20151224_003320	(The writings are hard to make out since they are smudged from the rain.)
```

Comment:
i'm unsure of the context, but if its referring to a collection of writings (like books for example) this is fine; otherwise, it should probably say "The writing is hard to make out"